integration/delete_test.go

```diff
@@ -55,3 +56,22 @@ func TestDeleteSimple(t *testing.T) {
 		})
 	}
 }
+
+func TestDeleteLimitNotSet(t *testing.T) {
```

Comment:
Do we want to add other cases like pass valid values, pass `float64`?

Reply:
We could. I'm going to improve that then